packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -615,44 +616,44 @@ String _flutterClientId = ''; // dimension8 Flutter tool clientId.
 
 String get userAppType => _userAppType;
 
-set userAppType(String __userAppType) {
-  _userAppType = __userAppType;
+set userAppType(String userAppType) {
+  userAppType = userAppType;
```

Comment:
this should be `_userAppType = userAppType`. Here and for all of these below. These might all cause infinite loops as is.

Reply:
Created issue: https://github.com/dart-lang/linter/issues/4306